### PR TITLE
Redis user roles package

### PR DESCRIPTION
**Pull request by TimMikeladze**
This PR introduces the redis package for user roles. Please ignore the other files in this diff as they are part of the initial starting code. I've marked the files I need reviewed. Thanks.

### CODE REVIEW

File: packages/redis/src/index.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
+import Redis from 'ioredis';
```
Comment (TimMikeladze) [inline packages/redis/src/index.js]: Review this file.

File: packages/redis/src/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,273 @@
+import RedisDBDriver from './index';
```
Comment (TimMikeladze) [inline packages/redis/src/index.spec.js]: Review this file.

File: packages/redis/src/index.js [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+import shortId from 'shortid';
+
+export default class {
+  constructor(options) {
+    this.redis = new Redis(options);
+  }
+  async addUserToRoles(userId, roles, group) {
+    const roleIds = [];
+    let groupId;
+    await Promise.all(roles.map(async (role) => {
```
Comment (alexyuly) [inline packages/redis/src/index.js]: What about `try`/`catch`? Is there any error handling needed for these async calls?

Comment (TimMikeladze) [inline packages/redis/src/index.js]: No error handling here, the package that consumes this one and calls these methods handles catching any errors that arise here.

**Pull request merged**